### DISCUSSION TITLE
Unauthorized projects should not be listed [Dec 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4698] [Discord Threads]
> 🙋‍♂️ **Soldado**  
## is there a way to hide the projects from others users?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1042601182083944468/Screenshot_2022-11-17_at_00.42.52.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### tracking here   
#### https://github.com/nocodb/nocodb/issues/4265  
#  
> 🙋‍♂️ **Soldado**  
#### thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1042601182398521394)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
